src/components/App.tsx

```diff
@@ -42,6 +43,10 @@ const App = () => {
 
 	const messageEditorRef = React.useRef<MessageEditorMethods>(null);
 
+	React.useEffect(() => {
+		loader.config({ paths: { vs: '../../resources/vs' } });
```

Comment:
i think this path has to be absolute.

For some reason monaco still tries to fetch files via cdn. Can you check this problem? you can check it in network in scripts(js) tab.

Maybe we need to update monaco versions. Also make sure it works in production aswell